MANIFEST.in

```diff
@@ -48,6 +37,5 @@ recursive-include docs Makefile
 recursive-include misc *.py
 recursive-include misc *.rst
 recursive-include tests *.py
-
-# added by check_manifest.py
 recursive-include invenio_i18n *.po
```

Comment:
just a question: shall we include also `*.mo` files (the compiled version of `*.po`).